### PR TITLE
auto insert css into head tag

### PR DESCRIPTION
**Pull request by xiaofuzi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,13 @@
...
   },
   "dependencies": {
-    "node-sass": "^3.4.2",
+    "ava": "^0.16.0",
+    "babel-preset-es2015-rollup": "^1.2.0",
+    "node-sass": "^3.8.0",
     "rollup-pluginutils": "^1.3.1"
   },
   "devDependencies": {
+    "ava": "^0.16.0",
```
Comment (adamdicarlo) [inline package.json:36]: `babel-preset-es2015-rollup` is already in `devDependencies`, and `ava` is added to _both_ devDeps and deps in this PR.

File: src/style.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    if(!css) return ;
+
+    if(typeof(window) == 'undefined') return ;
+    let style = document.createElement('style');
+    style.setAttribute('media', 'screen');
+
+    style.innerHTML = css
+    document.head.appendChild(style);
+    return css
+}
```
Comment (adamdicarlo) [inline src/style.js:16]: looks like this project uses semicolons, so you might want to add them in everywhere in this PR... also I'd change the spacing to match the project (space after `if`, no spaces in param list parens, etc).

**Comment by adamdicarlo** [comment]
Should this functionality be opt-in? I'm thinking that, especially if you tell the plugin to output compiled CSS to a file, you wouldn't want to include the CSS in your JS bundle and insert it into the head? (I might be wrong, I haven't actually used this plugin yet! 😄 )

**Comment by differui** [comment]
@xiaofuzi Thanks for this functionality.This is the first PR for my own project.😄

**Comment by differui** [comment]
@adamdicarlo The PR was merged.I will change semicolons and spaces to adapt to this project.

**Comment by differui** [comment]
@adamdicarlo Merged! I will add semicolons and change the spacing to match the project.

**Comment by xiaofuzi** [comment]
nice.

**Comment by differui** [comment]
Package was updated [rollup-plugin-sass@0.2.0](https://www.npmjs.com/package/rollup-plugin-sass).

**Pull request merged**